keras_cv/utils/visualize.py

```diff
@@ -0,0 +1,67 @@
+import tensorflow as tf
+import PIL.ImageColor as ImageColor
+
+
+def draw_masks_on_images(image, mask, color="red", alpha=0.4):
```

Comment:
can you include an example.  `examples/visualization/segmentation_mask.py`

Reply:
done